src/async/async-await.md

```diff
@@ -0,0 +1,52 @@
+# `async`/`await`
+
+At a high level, async Rust code looks very much like "normal" sequential code:
+
+```rust,editable
+use tokio::time;
```

Comment:
Is it possible to skip using `tokio` in this first example, and use futures and `block_on`?

Reply:
`futures` does not provide any time-based waking functionality. The only functionality in `futures` that would be non-trivial would be to send data over a channel, but I want to keep this example simple and parallel to the threads example.

Using `futures` for one of the later examples is a good idea, though.